app/controllers/DigitalSubscription.scala

```diff
@@ -118,20 +120,24 @@ class DigitalSubscription(
 
         type ApiResponseOrError[RES] = EitherT[Future, CreateDigitalSubscriptionError, RES]
 
-        if (validationPasses(request.body)) {
+        val normalisedTelephoneNumber = NormalisedTelephoneNumber.fromStringAndCountry(request.body.telephoneNumber, Some(request.body.country))
```

Comment:
Why does `request.body.country` need to be wrapped in Some here?

Reply:
That's a good point! It doesn't. That ended up there because I just brought over some code for NormalisedTelephoneNumber directly from membership-common. I've modified the `fromStringAndCountry` to take a `Country` rather than an `Option[Country]`